compiler/executable/match_/planner/function_plan.rs

```diff
@@ -24,9 +35,21 @@ impl ExecutableFunctionRegistry {
         Self { schema_functions, preamble_functions }
     }
 
-    pub(crate) fn empty() -> Self {
+    pub fn empty() -> Self {
         Self::new(Arc::new(HashMap::new()), HashMap::new())
     }
+
+    pub fn get(&self, function_id: FunctionID) -> &ExecutableFunction {
+        match &function_id {
+            FunctionID::Schema(id) => self.schema_functions.get(id).unwrap(),
+            FunctionID::Preamble(id) => self.preamble_functions.get(id).unwrap(),
+        }
+    }
+
+    // TODO: Find all references, update them and remove this.
+    pub fn TODO__empty() -> Self {
```

Comment:
Hygene clean up

Reply:
Will go in the next PR when fetch have their functions.